USE_CASES.md

```diff
@@ -205,3 +206,106 @@ The file `php.ini` should contain:
 google_app_engine.enable_curl_lite = 1
 ```
 
+<a name="personalization"></a>
+# Personalized Emails
+
+```php
+<?php
+/*
+This program demostrates how to send a personalized email
+to multiple recipients using the V3 SDK
+*/
+
+require "../SendGridV3B/sendgrid-php.php";
+
+/*
+define  SG_APIKEY and SG_FROM
+or you can create your own instead as follows
+*/
+
+define('SG_APIKEY',"Your api key");
```

Comment:
Please use this instead:`$apiKey = getenv('SENDGRID_API_KEY');` to promote not including a hard coded API Key in the code. Thanks!